### ISSUE TITLE
Clickable monitor name in status page

### ISSUE THREAD

**Issue by AlphaJack** [louislam/uptime-kuma#1885]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

It would be great to add an optional "public URL" field, to allow users to open the app by simply clicking on the monitor name in the status page

### ✔️ Solution

Add an optional field to make the monitor clickable

### ❓ Alternatives

Monitors remain unclickable

### 📝 Additional Context

Some monitors may be for internal use only or not offer a link at all

**Comment by Computroniks** [comment]
This has been implemented in #1741 to use the URL field for selected monitors

**Comment by Computroniks** [comment]
This will be in the next release

**Comment by AlphaJack** [comment]
Ok, thank you

**Comment by milindpatel63** [comment]
I saw that this feature was added quite a while back, and I am surprised no one said this till now,

The URL for the clickable link is the same URL that is used to monitor. For all of my services, I use a healthcheck endpoint as the URL, and this causes that healthcheck endpoint to be that clickable URL.

I thought it should be obvious to define a second public URL as a link to that service just for this feature.
Would love to see this implemented in future.
